### ISSUE TITLE
Limit for Maximum Concurrent Queries

### ISSUE THREAD

**Issue by PStump** [doitintl/bigquery-grafana#71]
#### Is your feature request related to a problem? Please describe.
We have switched from BigQuery to Bigtable in our project and we use BigQuery to access Bigtable. After adding multiple BigQuery queries, following error occurs.
> jobRateLimitExceeded: Job exceeded rate limits: Your project exceeded quota for concurrent queries that read Cloud Bigtable data sources. For more information, see https://cloud.google.com/bigquery/troubleshooting-errors

#### Describe the solution you'd like
Ability to set limit for maximum concurrent queries for the data source.

#### Describe alternatives you've considered
Add fewer queries (4) to one dashboard (limited amount of information per dashboard).

#### Teachability, Documentation, Adoption, Migration Strategy
If the default limit for maximum concurrent queries is set to infinite, there is no negative impact for current users.

**Comment by avivl** [comment]
@PStump Since the queries are generated on the client side, even if we limit them then another user will be able to open a browser and therefore bypass the limit

**Comment by spark2ignite** [comment]
@PStump you can request from Google to increase your concurrent queries from standard 50 concurrent queries to higher number. FYI.

**Comment by PStump** [comment]
Thank you both for the very fast answering!
@avivl you are absolutely right, no sense for this change.
@spark2ignite It is even worse, using Bigtable as an external source, there is a limit of 4 concurrent queries.

In the meantime, we have tried to solve this not plugin related, issue. And I would have liked to present an elegant solution to this problem for reference. Unfortunately, we got non.

Anyway, thank you for this great plugin!
